packages/bsky/src/db/migrations/index.ts

```diff
@@ -21,4 +21,5 @@ export * as _20230627T212437895Z from './20230627T212437895Z-optional-handle'
 export * as _20230629T220835893Z from './20230629T220835893Z-remove-post-hierarchy'
 export * as _20230703T045536691Z from './20230703T045536691Z-feed-and-label-indices'
 export * as _20230720T164800037Z from './20230720T164800037Z-posts-cursor-idx'
+export * as _20230802T232331476Z from './20230802T232331476Z-native-notifications'
```

Comment:
(before merge, we need to re-create the migration & ensure that it's the highest value in this list)

Reply:
Sounds good. I'll let you handle that since you will be the one merging this.